conf.py

```diff
@@ -37,6 +37,10 @@
     "source_branch": "main",
 }
 html_static_path = ['_static']
+html_extra_path = [
+    'include/release-cycle.svg',
+    'include/release-cycle-all.svg',
+]
```

Comment:
Why is this needed? Semantically, I think we should prefer static path to extra path: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path

> A list of paths that contain extra files not directly related to the documentation

A

Reply:
> Why is this needed? 

So that they're served at https://devguide.python.org/release-cycle.svg


> Semantically, I think we should prefer static path to extra path: [sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path)
> 
> > A list of paths that contain extra files not directly related to the documentation
> 
> A

Okay, so have the generate script save them to _static, and have them served at https://devguide.python.org/_static/release-cycle.svg ?

I've pushed a commit for this.